### PR TITLE
fix issue found for Set-AzVmssVMRunCommand

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description

New parameter sets for Set-AzVMRunCommand and Set-AzVmssVMRunCommand were added in this PR(https://github.com/Azure/azure-powershell/pull/19935) set to GA on 11/1, 
but the feature owner @vivlingaiah has already found a bug(https://github.com/Azure/azure-powershell/issues/19985), and I want to address them before it goes out. 

Also adding examples here instead of in the PR that @vivlingaiah made in this PR: https://github.com/Azure/azure-powershell/pull/19969

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
@dingmeng-xue , @VeryEarly  pls see the description.  The PR I made last week for this update has a bug and I want to address it before it goes out

**Comment by VeryEarly** [comment]
please create OOB request, it's too late for current release

**Comment by Nickcandy** [comment]
It looks good to me. Please help me double check @BethanyZhou

File: src/Compute/Compute.Autorest/docs/Set-AzVMRunCommand.md [hunk 1]
```diff
@@ -37,18 +37,59 @@ The operation to create or update the run command.
...
 ## EXAMPLES
 
-### Example 1: Simple Example
+### Example 1: Create or update Run Command on a VM using a storage blob SAS URL
 ```powershell
-Set-AzVMRunCommand -ResourceGroupName $rgname -VMName $vmname -RunCommandName 'firstruncommand' 
+Set-AzVMRunCommand -ResourceGroupName MyRG0 -VMName MyVMEE -RunCommandName MyRunCommand -Location EastUS2EUAP -SourceScriptUri "https://myst.blob.core.windows.net/mycontainer/myscript.ps1?sp=r&st=2022-10-27T21:02:35Z&se=2022-10-28T05:02:35Z&spr=https&sv=2021-06-08&sr=b&sig=0I%2FIiYayRwHasfasasfdasdfasdeTsQjLnpZjA%3D"
 ```
 
-```output
```
Comment (BethanyZhou) [inline src/Compute/Compute.Autorest/docs/Set-AzVMRunCommand.md:45]: Please do not remove output block,

File: src/Compute/Compute.Autorest/examples/Set-AzVmssVMRunCommand.md [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
+```powershell
+Set-AzVmssVMRunCommand -ResourceGroupName MyRG0 -VMScaleSetName MyVMSS -InstanceId 0 -RunCommandName MyRunCommand -Location EastUS2EUAP -SourceCommandId DisableWindowsUpdate
+```
+
+Create or update Run Command on a VMSS VM instance using pre-existing commandId. Available commandIds can be retrieved using Get-AzVMRunCommandDocument.
+
+### Example 5: Create or update Run Command on a VMSS VM instance and stream standard output and standard error messages to output and error Append blobs.
+```powershell
+Set-AzVmssVMRunCommand -ResourceGroupName MyRG0 -VMScaleSetName MyVMSS -InstanceId 1 -RunCommandName MyRunCommand3 -Location EastUS2EUAP -ScriptLocalPath "C:\MyScriptsDir\MyScript.ps1" -OutputBlobUri "https://vivst.blob.core.windows.net/vivcontainer/output.txt?sp=racw&st=2022-10-27T22:18:36Z&se=2022-10-28T06:18:36Z&spr=https&sv=2021-06-08&sr=b&sig=HQAu3Bl%2BKMofYTjMo8o5hasfadsfasdF4jIkRJra4S5FlEo%3D" -ErrorBlobUri "https://vivst.blob.core.windows.net/vivcontainer/error.txt?sp=racw&st=2022-10-27T22:18:36Z&se=2022-10-28T06:18:36Z&spr=https&sv=2021-06-08&sr=b&sig=HQAu3Bl%2BKMofYTjMo8o5h%asfasdfgdT%2F4jasfasdf5FlEo%3D"
```
Comment (BethanyZhou) [inline src/Compute/Compute.Autorest/examples/Set-AzVmssVMRunCommand.md:31]: Please add output block if applicable.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
Looks good to me with few changes required

**Comment by grizzlytheodore** [comment]
@BethanyZhou  added back output blocks. Please review/merge !

**Comment by grizzlytheodore** [comment]
@Nickcandy  looks all good. Please review/merge! thank you

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
Please add output for each example.

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Comment by BethanyZhou** [comment]
Hi @Nickcandy , could you help carry @grizzlytheodore 's changes to main branch?

**Comment by Nickcandy** [comment]
@BethanyZhou Merged

**Pull request merged**